visual-e2e/examples/test/configs/wdio.shared.conf.js

```diff
@@ -1,23 +1,48 @@
+
+
 exports.config = {
     // ====================
     // More information about the configuration is found here https://webdriver.io/docs/configurationfile.html
     // ====================
     runner: 'local',
     specs: [
-        './test/specs/**/*.js'
+        './test/specs/simple.spec.js'
     ],
     maxInstances: 100,
-    logLevel: 'debug',
+    logLevel: 'warn',
     bail: 0,
     baseUrl: 'https://www.saucedemo.com/',
-    waitforTimeout: 10000,
-    connectionRetryTimeout: 90000,
+    waitforTimeout: 30000,
+    connectionRetryTimeout: 120000,
     connectionRetryCount: 3,
     reporters: ['spec'],
     services: [],
     //Screener Configuration
     hostname: 'hub.screener.io',
     port: 443,
     protocol: 'https',
-    path: '/wd/hub'
+    path: '/wd/hub',
+    capabilities: [
```

Comment:
Why do you have the caps here and the cross platform in a separate file? My advise would be that if you have more than one example of different caps that you use a shared conf file with only the main config and then separate config files with only the differences per config file as we have for the rest of the examples in this repo